feat(baremetal): review docs (#3561)
scaleway/docs-content: bare-metal/elastic-metal/how-to/add-virtual-mac-address.mdx

---
meta:
  title: How to add a virtual MAC address to a flexible IP
  description: Learn how to add a virtual MAC address to a flexible IP for Scaleway Elastic Metal servers.
content:
  h1: How to add a virtual MAC address to a flexible IP
  paragraph: This guide explains how to add a virtual MAC address to a flexible IP for Scaleway Elastic Metal servers.
tags: virtual-mac, virtual-mac-address, flexible-ip
dates:
  validation: 2024-08-05
  posted: 2021-05-26
categories:
  - bare-metal
---

Virtual MAC addresses act as virtual interface identifiers necessary for virtualization. When running multiple virtual machines on a single physical Elastic Metal server, you can configure the [MAC address](https://en.wikipedia.org/wiki/MAC_address) of the virtual network interface to match the virtual MAC associated with the [flexible IP](/bare-metal/elastic-metal/concepts/#flexible-ip), ensuring network connectivity for the virtual machine.

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)

> - [Owner](/identity-and-access-management/iam/concepts/#owner) status or appropriate [IAM permissions](/identity-and-access-management/iam/concepts/#permission) for actions in the intended Organization

Reviewer: ```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```
